lib/openstudio-standards/standards/Standards.Space.rb

```diff
@@ -1753,6 +1753,311 @@ def space_design_internal_load(space)
+          ltg_values[ihr] += ltg_w * ltg_sch_value
+        end
+      end
+    end
+
+    # Luminaire Objects
+    space.spaceType.get.luminaires.each do |light|
+      ltg_sch_values = nil
+      ltg_sch = light.schedule
+      ltg_w = light.lightingPower(space.floorArea, ppl_total)
+      # not sure if above line is valid, so calculate from parts instead until above can be verified
+      ltg_w = light.getPowerPerFloorArea(space.floorArea) * space.floorArea
+      ltg_w += light.getPowerPerPerson(ppl_total) * ppl_total
+
+      if light.isScheduleDefaulted
+        # Check default schedule set
+        unless (space.spaceType.get.defaultScheduleSet.empty?)
+          unless space.spaceType.get.defaultScheduleSet.get.lightingSchedule.empty?
+            ltg_sch = space.spaceType.get.defaultScheduleSet.get.lightingSchedule
+          end
+        end
+      end
+      if ltg_sch.is_initialized
+        ltg_sch_obj = ltg_sch.get
+        ltg_sch_values = get_8760_values_from_schedule(model, ltg_sch_obj)
+        if !ltg_sch_values.nil?
+          ltg_sch_max = ltg_sch_values.max
+        else
+          OpenStudio.logFree(OpenStudio::Warn, 'openstudio.standards.Space', "Failed to retreive lighting schedule for luminaires for #{space.name}.  Assuming #{ltg_w} W.")
+        end
+      end
+
+      if !ltg_sch_values.nil?
+        ltg_sch_value = 1.0
+        (0..8759).each do |ihr|
+          ltg_sch_value = ltg_sch_values[ihr] unless ltg_sch_ruleset.nil?
+          ltg_values[ihr] += ltg_w * ltg_sch_value
+        end
+      end
+    end
+
+    # Equipment Loads
+    eq_w = space.electricEquipmentPower
+    space.spaceType.get.electricEquipment.each do |equip|
+      eqp_type = 'electric equipment'
+      load_values = space_get_equip_annual_array(model, space, equip, eqp_type, ppl_total, load_values)
+    end
+    space.spaceType.get.gasEquipment.each do |equip|
+      eqp_type = 'gas equipment'
+      load_values = space_get_equip_annual_array(model, space, equip, eqp_type, ppl_total, load_values)
+    end
+    space.spaceType.get.steamEquipment.each do |equip|
+      eqp_type = 'steam equipment'
+      load_values = space_get_equip_annual_array(model, space, equip, eqp_type, ppl_total, load_values)
+    end
+    space.spaceType.get.hotWaterEquipment.each do |equip|
+      eqp_type = 'hot water equipment'
+      load_values = space_get_equip_annual_array(model, space, equip, eqp_type, ppl_total, load_values)
+    end
+    space.spaceType.get.otherEquipment.each do |equip|
+      eqp_type = 'other equipment'
+      load_values = space_get_equip_annual_array(model, space, equip, eqp_type, ppl_total, load_values)
+    end
+
+    # Add lighting and people to the load values array
+    (0..8759).each do |ihr|
+      load_values[ihr] += ppl_values[ihr] + ltg_values[ihr]
+    end
+
+      return load_values
+  end
+
+  # @author: Doug Maddox, PNNL
+  # @param: model [Object]
+  # @param: fan_schedule [Object]
+  # @return: 
```

Comment:
We probably want to complete the description of the method.

Reply:
I changed the format, @dmaddoxwhite , could you please update the description?